cmd/utils/flags.go

```diff
@@ -1169,6 +1170,12 @@ var (
 		Value:    false,
 		Category: flags.StateExpiryCategory,
 	}
+	StateExpiryPruneLevelFlag = &cli.UintFlag{
+		Name:     "state-expiry.prunelevel",
+		Usage:    "set prune level for state expiry,",
```

Comment:
small typo, with additional "," at the end.

The correct change would be: "set prune level for state expiry".